ngraph/frontend/paddlepaddle/src/op/stack.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <node_context.hpp>
+
+#include "default_opset.hpp"
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+using namespace default_opset;
+NamedOutputs stack(const NodeContext& node) {
+    auto datas = node.get_ng_inputs("X");
+    auto axis = node.get_attribute<int32_t>("axis", 0);
+    auto data_shape = datas[0].get_partial_shape();
+    auto data_type = datas[0].get_element_type();
+    OutputVector node_datas_reshape;
+
+    auto axis_const = std::make_shared<Constant>(element::i64, Shape{}, axis);
```

Comment:
add range check for `axis` to ensure it locates inside  `[−(R+1),R+1]`, where `R` is the dimension of the first input

Reply:
done